lib/lightning/projects.ex

```diff
@@ -232,6 +230,7 @@ defmodule Lightning.Projects do
       {:ok, string}
 
   """
+  @spec export_project(:yaml, any) :: {:ok, binary}
```

Comment:
@wanecode I believe the convention is to have the `@spec` between the doc tag and the `def`.